### PR TITLE
docs: Switch to Ethical Ads

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

- copy of https://github.com/eslint/eslint.org/pull/522
- adds ethical ads

#### Is there anything you'd like reviewers to focus on?
nope

### CODE REVIEW

File: docs/src/assets/scss/ads.scss
Comment (harish-sethuraman) [inline docs/src/assets/scss/ads.scss:7]: we had a wrapper in the main page to avoid shift of content when added popsup. Seems like docs didn't have it so added this.

File: docs/src/_includes/partials/ads.html
Comment (nzakas) [inline docs/src/_includes/partials/ads.html:1]: Should be just `ad.html` (singular).

File: docs/src/_data/sites/en.yml
Comment (nzakas) [inline docs/src/_data/sites/en.yml:29]: We should keep `serve:""` and `placement:""` here to make it easier to switch back (and remember what was required)

Comment (harish-sethuraman) [reply docs/src/_data/sites/en.yml]: Changed 👍🏻

File: docs/src/assets/scss/ads.scss
Comment (nzakas) [inline docs/src/assets/scss/ads.scss:140]: We can delete this.
```suggestion
// [data-ea-publisher].loaded .ea-content a:hover,
// [data-ea-type].loaded .ea-content a:hover {
//     text-decoration: none;
//     color: var(--link-color);
// }
```

**Comment by nzakas** [comment]
We are ready to roll this out pending the removal of the unnecessary comments.

**Comment by nzakas** [comment]
Once merged, we should also merge this: https://github.com/eslint/eslint.org/pull/522

File: docs/src/assets/scss/ads.scss
Comment (harish-sethuraman) [reply docs/src/assets/scss/ads.scss]: done

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

I'm leaving this open if @nzakas wants to take another look before merging, and because ideally this should be merged just before we start today's release so that the docs sites are updated around the same time (`eslint.org/docs/head` right before and `eslint.org/docs/next` right after the release).

What about `eslint.org/docs/latest`? Looks like we should prepare another PR that should target the `latest` branch?

**Review by nzakas — APPROVED** [review]
LGTM. I'll let @mdjermanovic merge just before the release.

**Comment by mdjermanovic** [comment]
Since we've already merged https://github.com/eslint/eslint/pull/18102 and the new ads are live on `eslint.org/docs/latest`, we could as well merge this one now? :)

**Comment by nzakas** [comment]
Oops, yes. :)

**Pull request merged**
